docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -377,9 +303,8 @@ SELECT (organization).office FROM users
 NOTE: When selecting the entire object, the query will always try to return the underlying object verbatim.
 For Java Types, this means returning an underlying Java class instance, which can fail with a `ClassNotFoundException`
 if the class is not in the classpath of the client (or embedded server) JVM.
-A way to avoid this is to select field by field instead. Additionally, this issue is not relevant for Compact
-and Portable types as sub-objects in these mappings and types are of `GenericRecord` subclass;
-`PortableGenericRecord` and `CompactGenericRecord` are present in the base distribution of Hazelcast.
+To avoid this, you can select a field-by-field instead. Additionally, this issue does not apply to Portable, Compact, or Avro types as sub-objects in these mappings and types are `GenericRecord` subclasses;
```

Comment:
remeber that there are different clients that java. In particular will Avro be supported in them?

Reply:
fef8c8c2